<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chart</class>
 <widget class="QWidget" name="chart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart</string>
  </property>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>69</y>
     <width>571</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="condBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>恒定转速</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>恒定功率</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>条件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索螺旋桨</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeBox">
   <property name="geometry">
    <rect>
     <x>458</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>拉力曲线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>效率曲线</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>功率曲线</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="typeLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>纵坐标</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>81</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="condLabel">
      <property name="text">
       <string>转速/RPM</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="condInput"/>
    </item>
    <item>
     <widget class="QLabel" name="propLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
